### ISSUE TITLE
Shared Boards - Allow to open modal for more information

### ISSUE THREAD

**Issue by AllanOcelot** [agileseason/agileseason#160]
On a shared board, example ( https://agileseason.com/#/shared/board/4b4e403d6ea0a182f6994b8c67d2ec55 )  - a user visiting is unable to get more information on an item. 

Expected Behavior: 
 - user visited shared board via URL
 - user clicks on 'ticket'
 - ticket opens modal, with body information text present as on original ticket. 

Current Behavior: 
 - user visits shard board via URL
 - user clicks on ticket
 - no action

Alternative / Workaround Behavior
 - IF unable to get ticket information for non-signed-in users
 - SHOULD open original issue in gituhub, in new browser tab

**Comment by blackchestnut** [comment]
@AllanOcelot hmm... it's possible, but only for issues' body, not for comments. Would it be useful?

**Comment by AllanOcelot** [comment]
Yes, that would be great - comments should only be for team members I think. 

If we can show the body, it makes shareable links very useful for roadmaps on projects, thank you!
